README.md

```diff
@@ -89,6 +89,29 @@ poetry install
 poetry shell
  ```
 
+### Pre-Commit Hooks
+We are using [Pre-Commit](https://pre-commit.com/) to enforce formatting, lint rules, and code analysis so that
```

Comment:
This is great documentation, very nice. The local install instructions were very much appreciated.